iothub/device/samples/getting started/RoutingTutorial/SimulatedDevice/Parameters.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using CommandLine;
+
+namespace SimulatedDevice
+{
+    /// <summary>
+    /// Parameters for the application.
+    /// </summary>
+    internal class Parameters
+    {
+        [Option(
+            'u',
+            "IoTHubUri",
+            Required = true,
+            HelpText = "The URI for the IoT hub.")]
+        public string IotHubUri { get; set; }
+
+        [Option(
+            'd',
+            "DeviceId",
```

Comment:
Instead of hostname, device id and device key, why not just use device connection string?